lldb/packages/Python/lldbsuite/test/lldbutil.py

```diff
@@ -1654,6 +1654,26 @@ def find_library_callable(test):
     )
 
 
+def target_install(test, filename=None, path=None):
```

Comment:
This is optional, but I think something like `install_to_target` would be a better match for the naming style of the surrounding functions.

Reply:
Updated. Thanks.